### PR TITLE
fix(ui): Add new alert status icons

### PR DESCRIPTION
**Pull request by scttcper**
These icons appear in both the new alert header inside a button, and in the alerts list.

This PR also restyles the status dropdown on the alert detail page.

before:
image

new icons:
image

new alerts header:
image

### CODE REVIEW

File: src/sentry/static/sentry/app/views/alerts/status.tsx [hunk 1]
```diff
@@ -2,62 +2,49 @@ import React from 'react';
...
   const isResolved = status === IncidentStatus.CLOSED;
   const isWarning = status === IncidentStatus.WARNING;
 
-  const icon = isResolved
-    ? 'icon-circle-check'
-    : isWarning
-    ? 'icon-warning-sm'
-    : 'icon-circle-exclamation';
+  const icon = isResolved ? (
+    <IconCheckmark color="green400" />
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/views/alerts/status.tsx:21]: 😍

File: src/sentry/static/sentry/app/views/alerts/status.tsx [hunk 2]
```diff
@@ -2,62 +2,49 @@ import React from 'react';
...
-    return theme.yellowDark;
-  }
-
-  return theme.redDark;
-}
-
-const Wrapper = styled('div')<{isSmall: boolean}>`
-  display: flex;
+const Wrapper = styled('div')`
+  display: grid;
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/views/alerts/status.tsx:41]: Thanks for using grid. Can you use grid gap to get rid of the margins on the icon?

**Review by evanpurkhiser — CHANGES_REQUESTED** [review]
Just my comment about grid gap

**Comment by scttcper** [comment]
Updated the dropdown after the design discussions cc @evanh 

image
Note: this is without the new checkmark icon.

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**